code/controllers/master_controller.dm

```diff
@@ -36,29 +36,7 @@ var/global/pipe_processing_killed = 0
 	return QDEL_HINT_HARDDEL_NOW
 
 /datum/controller/game_controller/proc/setup()
-	preloadTemplates()
-	if(!config.disable_away_missions)
-		createRandomZlevel()
-	// Create 6 extra space levels to put space ruins on
-	if(!config.disable_space_ruins)
-		var/timer = start_watch()
-		log_startup_progress("Creating random space levels...")
-		seedRuins(level_name_to_num(EMPTY_AREA), rand(0, 3), /area/space, space_ruins_templates)
-		log_startup_progress("Loaded random space levels in [stop_watch(timer)]s.")
-
-		// load in extra levels of space ruins
-
-		var/num_extra_space = rand(config.extra_space_ruin_levels_min, config.extra_space_ruin_levels_max)
-		for(var/i = 1, i <= num_extra_space, i++)
-			var/zlev = space_manager.add_new_zlevel("[EMPTY_AREA] #[i]", linkage = CROSSLINKED)
-			seedRuins(zlev, rand(0, 3), /area/space, space_ruins_templates)
-
-	space_manager.do_transition_setup()
-
 	setupfactions()
 	setup_economy()
 
-	for(var/i=0, i<max_secret_rooms, i++)
-		make_mining_asteroid_secret()
-
 	populate_spawn_points()
```

Comment:
Oh, Master controller, you used to run everything....now uh, you're probably going to get killed off this year.

Reply:
I wonder how Goon feels that their fancy-shmancy process scheduler is getting killed off by tg's thingamabob

Reply:
To be honest, the biggest argument for Goon going open source is so they can use StonedMC, IMO.

Then they don't have to do anything silly like their "real time reagent/breathing/mob processing".